src/payment/strategies/googlepay/googlepay-payment-strategy.spec.ts

```diff
@@ -291,6 +291,32 @@ describe('GooglePayPaymentStrategy', () => {
                 },
             });
 
+            await strategy.initialize(googlePayOptions);
+            await expect(strategy.execute(getGoogleOrderRequestBody())).rejects.toThrow(MissingDataError);
+            expect(googlePayPaymentProcessor.displayWallet).toBeCalled();
+        });
+
+        it('gets again the payment information and get a new nonce', async () => {
```

Comment:
I don't see a rejected promise or any hint that this is checking for a failed payment method as the PR suggests. Is the test checking for the correct thing?